join-chat-to-teams-meeting/Chat Experience with File Sharing/README.md

```diff
@@ -0,0 +1,45 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+- azure-communication-common
+- azure-communication-chat
+- azure-communication-calling
+- azure-communication-identity
+---
+
+# Teams Interop Chat App with File Attachment Support
+
+This code sample is explained as part of [this tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/chat-interop/meeting-interop-features-file-attachment). Please make sure you have gone through it first.
+
+## Prerequisites
+- An active Communication Services resource. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource).
+- A [Teams deployment](https://docs.microsoft.com/deployoffice/teams-install)
+
+## Code Structure
+
+- **./index.html:** to configure a basic layout that will allow the user to join a teams meeting.
+- **./client.js:** contain the application logic.
+
+## Setup
+
+1. Clone this sample
+3. Run `npm install`
+4. Grab the connection string from your communication resource and replace it at line #36 in client.js ```const connectionString = "<SECRET_CONNECTION_STRING>";```.
+6. Similarly, replace the endpoint URL at line #37 in client.js ```const endpointUrl = "<ENDPOINT_URL>";```.
```

Comment:
nitpick: The numbers here shouldn't really matter since `.md` will automagically number them properly. But ideally you should either number is properly or change them all to `1.`

Reply:
good catch! the numbering does look ridiculous lol